apps/console/src/features/connections/components/edit/connection-edit.tsx

```diff
@@ -451,8 +452,8 @@ export const EditConnection: FunctionComponent<EditConnectionPropsInterface> = (
         const isAttributesEnabledForOIDC: boolean = isOidc && UIConfig?.isCustomClaimMappingEnabled;
 
         // Evaluate whether to Show/Hide `Attributes`.
-        if ((attributesForSamlEnabled || shouldShowTab(type, ConnectionTabTypes.USER_ATTRIBUTES))
-            && (IdentityProviderManagementConstants.IDP_TEMPLATE_IDS.OIDC === type && isAttributesEnabledForOIDC)
+        if (((attributesForSamlEnabled || shouldShowTab(type, ConnectionTabTypes.USER_ATTRIBUTES))
+            || (IdentityProviderManagementConstants.IDP_TEMPLATE_IDS.OIDC === type && isAttributesEnabledForOIDC))
```

Comment:
Shall we move this `type === template_ids.OIDC` check to `isAttributesEnabledForOIDC` variable for better readability?

```js
const isAttributesEnabledForOIDC: boolean = isOidc 
    && type === IdentityProviderManagementConstants.IDP_TEMPLATE_IDS.OIDC
    && UIConfig?.isCustomClaimMappingEnabled;

```